test/integration/controllers/projects/add.js

```diff
@@ -1,2 +1,49 @@
 'use strict';
+var chai = require('chai');
+var chaiHttp = require('chai-http');
+var expect = require('chai').expect;
+chai.use(chaiHttp);
 
+var manager = require('../../../manager');
+
+
+describe('testing projects.js', function () {
+	this.timeout(10000);
+	before(manager.setUp);
+	after(manager.tearDown);
+
+	it('add a project', function(done){
+		before(manager.createUser('Testuser', 'email@email.com', 'password'));
+
+		var agent = chai.request.agent('http://localhost:3000/')
+		agent
+			.post('login')
+			.set('content-type', 'application/x-www-form-urlencoded')
+			.send({
+				username : 'Testuser',
+				password : 'password'
+			})
+			.then(function (res) {
+				// The `agent` now has the sessionid cookie saved, and will send it
+				// back to the server in the next request:
+				return agent.post('projects/add')
+				.set('content-type', 'application/x-www-form-urlencoded')
+				.send({
+					name : 'test',
+					description : 'test',
+					amount : '12345678.12345678',
+					token : '1BHPGY7Rb9WaBBkYPKjZTnKYRzt5mC8NPM'
+				})
+				.end(function(err, res){
+					expect(res).to.have.status(200);
+					expect(res.body).to.be.a('object');
+					expect(res).to.redirectTo('http://localhost:3000/');
+					done();
+				})
+			})
+			.catch(function(error){
+				console.log(error);
+			})
```

Comment:
Should use:
```js
.catch(done);
```
So that the test will fail if there is a database error, instead of timing out.